third_party/material/app.scss

```diff
@@ -9,3 +9,5 @@
 @use '@material/ripple/mdc-ripple';
 @use '@material/select/mdc-select';
 @use '@material/textfield/mdc-text-field';
+@use "@material/fab";
+@include fab.core-styles;
```

Comment:
Is this needed? The reference file uses only fab:
https://github.com/material-components/material-components-web/blob/master/packages/material-components-web/material-components-web.scss#L35

Reply:
It seems not when `/mdc-fab` is added in the end of the `@use`. 

In https://material.io/components/buttons-floating-action-button/web#using-fabs it said 
```
@use "@material/fab";

@include fab.core-styles;
```

But ``` @material/fab/mdc-fab ``` seems to work so let's keep that. :-)